kratos/tests/cpp_tests/geometries/test_line_2d_2.cpp

```diff
@@ -270,6 +270,48 @@ namespace Testing {
         KRATOS_EXPECT_NEAR(test_point_local_coords(2), 0.0, TOLERANCE);
     }
 
+    /** Checks the point local coordinates for a given point respect to the line.
```

Comment:
It seems as if one or a few words are missing in this sentence... I have problems reading and understanding it (around the word "respect"). Is it just me, or should the sentence be improved?

The same applies to the sentence on line 295.

Reply:
changed it to 'with respect to'